deployer/README.md

```diff
@@ -385,6 +404,9 @@ processed. Note: this means that aggregated logs for the configured
 projects could fall behind and even be lost if the pod were deleted
 before Fluentd caught up.
 
+*NOTE* Throttling does not work when using the systemd journal as the
```

Comment:
Would prefer a little more detail here... e.g. "Throttling relies on the use of Docker json-log files and will not be active when using the systemd journal as the log source."